system-int-test/test/cmr/system_int_test/search/collection_shapefile_search_test.clj

```diff
@@ -89,49 +97,49 @@
         esri-point (make-coll :geodetic "esri-point" (p/point -80 35))]
     (index/wait-until-indexed)
 
-    (testing "Search by ESRI shapefile"
-      (are3 [shapefile items]
-            (let [found (search/find-refs-with-multi-part-form-post
-                         :collection
-                         [{:name "shapefile"
-                           :content (io/file (io/resource (str "shapefiles/" shapefile)))
-                           :mime-type "application/shapefile+zip"}
-                          {:name "provider"
-                           :content "PROV1"}])]
-              (d/assert-refs-match items found))
+    (doseq [fmt (keys formats)
+            :let [{extension :extension mime-type :mime-type} (get formats fmt)]]
+      (debug (format "%s %s %s" fmt extension mime-type))
```

Comment:
I think this debug can be removed with the change to the testing line.

Reply:
done